device/alpaka/src/utils/utils.hpp

```diff
@@ -40,7 +40,7 @@ static constexpr std::size_t warpSize =
 template <typename TAcc>
 inline WorkDiv makeWorkDiv(Idx blocksPerGrid,
                            Idx threadsPerBlockOrElementsPerThread) {
-    if constexpr (::alpaka::accMatchesTags<TAcc, ::alpaka::TagGpuCudaRt>) {
+    if constexpr (::alpaka::accMatchesTags<TAcc, ::alpaka::TagGpuCudaRt> || ::alpaka::accMatchesTags<TAcc, ::alpaka::TagGpuHipRt>) {
```

Comment:
To note, there is an upcoming Alpaka PR to help with this as well (https://github.com/alpaka-group/alpaka/pull/2263) which would have avoided this problem entirely.

Reply:
:+1: